spec/controllers/admin/targets_controller_spec.rb

```diff
@@ -0,0 +1,168 @@
+require 'spec_helper'
+
+describe Admin::TargetsController, type: :controller do
+  let(:admin) { create(:admin) }
+  let(:conference) { create(:conference) }
+  let(:target) { create(:target, conference: conference, target_count: 100) }
+
+  context 'user is admin' do
+    before { sign_in admin }
+
+    describe 'GET #index' do
+      before { get :index, conference_id: conference.short_title }
+
+      it 'renders index template' do
+        expect(response).to render_template('index')
+      end
+
+      it 'assigns target and conference variable' do
```

Comment:
assigns @targets and @conference variable**s**

The index action uses the variable in plural (@targets).
Both variables come from cancancan authorization in the controller.
https://github.com/openSUSE/osem/blob/master/app/controllers/admin/targets_controller.rb#L3
https://github.com/openSUSE/osem/blob/master/app/controllers/admin/targets_controller.rb#L4